core/src/nucleus/ribosome/api/remove_entry.rs

```diff
@@ -42,10 +42,14 @@ pub fn invoke_remove_entry(runtime: &mut Runtime, args: &RuntimeArgs) -> ZomeApi
         .block_on(get_entry_result_workflow(&context, &get_args));
 
     if let Err(_err) = maybe_entry_result {
+        // TODO: don't just return "Unspecified"
```

Comment:
@AshantiMutinta what exactly can this error be?

Reply:
Probably an error that tells us something went wrong during this get process. I'm not sure of the list of error types but I will look through and if not, we would have to create a new one for it.